server/mockData.js

```diff
@@ -89,7 +89,11 @@ const mockProcessData = {
           photo: { isWilling: "yes", images: [] },
           nextOfKin: { fullName: "njkl" },
           carer: { hasCarer: "no" },
-          otherSupport: { fullName: "mzsl", role: "." },
+          otherSupport: {
+            fullName: "mzsl",
+            role: ".",
+            phoneNumber: "07833109098",
```

Comment:
I know this is only mock data but I think the keys here should match what you have in the schema e.g. have `hasSignedUp` and `notes` - I'm not sure though...

Reply:
yep , I wasn't sure either way.